### ISSUE TITLE
Quantum - FuseBox.import() throws warnings

### ISSUE THREAD

**Issue by NGPixel** [fuse-box/fuse-box#604]
When using Quantum, any usage of `FuseBox.import()` results in a *Computed statement resolution* warning, even though the source is a static string.

### Example

The following code:
```js
FuseBox.import('/js/ace/ace.js', ace => {
   // code
})
```
results  in:
```markdown
Warnings:
Your quantum bundle might not work
  - Computed statement warning in default/something.js

  * Set { warnings : false } if you want to hide these messages
  * Read up on the subject http://fuse-box.org/page/quantum#computed-statement-resolution
```

**Comment by nchanged** [comment]
#605

**Comment by nchanged** [comment]
@NGPixel it will be resolved, currently Quantum doesn't support async loading. We need `import` functionality.

**Comment by nchanged** [comment]
Should be solved in `2.2.0-beta.6` with `import()` support
https://github.com/fuse-box/fuse-box-examples/blob/2.2.0/examples/code-splitting/src/index.ts#L1
